### ISSUE TITLE
[Consistency] User encounters design components that are inconsistent or confusing.

### ISSUE THREAD

**Issue by erinrwhite** [erinrwhite/stagingreview-mods#9]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Testing team
**Product:** Testing product
**Feature:** Testing feature

## Findings details
**VA.gov Experience Standard - issue:** User encounters design components that are inconsistent or confusing.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @erinrwhite (IA)

### Description

On the confirmation page, the heading hierarchy moves from the form title, an H1, to the ""Claim submitted"" header, an H3.

## Recommended action
Update the confirmation page to use an H2 for the ""Claim submitted"" header. The Help users keep a record of submitted information pattern might be helpful here.

## References

- **WCAG Success Criteria:** [WCAG 2.4.6 Headings and Labels - Level AA](https://www.w3.org/TR/WCAG21/#headings-and-labels)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern:** [Keep a record of submitted information](https://design.va.gov/patterns/help-users-to/keep-a-record-of-submitted-information)
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** Headings
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
